www/source/index.html.slim

```diff
@@ -59,7 +59,7 @@ section.home--hero
           li
             span.logo-icon.mesosphere title="Mesosphere"
 
-  .row.home--infographic-title
+  .row.home--infographic-title#home--infographic-title
```

Comment:
Since this will [go into a URL](https://www.hanselman.com/blog/URLsAreUI.aspx), how about something less obscure and more descriptive, like `#lifecycle-diagram`?

Reply:
In the paragraph above it, it is referred to as `reference diagram`. Shall we keep it consistent? `Lifecycle` reminds me of the software lifecycle management that the Supervisor does. The diagram here is mainly about a developer's workflow.

Reply:
@apriofrost Works for me! `reference-diagram` it is.